packages/headless/src/features/facets/facet-api/request.ts

```diff
@@ -87,23 +87,29 @@ export type FacetType =
   | 'numericalRange'
   | 'hierarchical';
 
+export type SortCriteriaType =
+  | 'score'
+  | 'alphanumeric'
+  | 'ascending'
+  | 'descending'
+  | 'occurrences'
+  | 'automatic'
+  | 'alphanumericDescending';
+
+export type SpecificSortCriteriaType = Exclude<
+  SortCriteriaType,
+  'ascending' | 'descending' | 'automatic' | 'alphanumericDescending'
+>;
+
```

Comment:
If i understood correctly, the SortCriteriaType will be removed in v3. Would it be possible to have a entire different type instead of relying on Exclude the soon to be removed type ?

Reply:
This lead me down a bit of a rabbit hole honestly. In short, we won't be removing the `SortCriteriaType` type... At least not _all of it_. Certain things, such as `automatic` and the duplicated range facet sort criterion (`ascending` and `descending`) will be used for the foreseeable future, so I think I'll take some time to strip these models down to be more obviously scoped so that when the time comes to pull the plug on the old type(s), it will be much more clear and simple 🙂